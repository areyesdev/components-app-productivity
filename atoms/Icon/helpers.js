const ICONS_ASSETS_PATH = '/icons',
  iconMap = {
    arrowRight: 'arrow-right',
    angleDown: 'angle-down',
    iconPlus: 'icon-plus',
  },
  iconSize = {
    sm: 15,
    md: 26,
    lg: 40,
    xl: 40,
  }

export const mapType = (type) => `${ICONS_ASSETS_PATH}/${iconMap[type]}.svg`
export const mapSize = (size) => iconSize[size]
