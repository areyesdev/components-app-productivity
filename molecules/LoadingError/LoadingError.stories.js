import { LoadingError } from '.'

import { getTemplate } from '../../helpers/storybook'

const Template = getTemplate(LoadingError)

export default {
  title: 'Molecules/LoadingError',
  component: LoadingError,
  args: {
    children:
      "Exile ruins brook Angmar inn wars retaken. Swiftly attacks Grma merely! Opinion Eorlingas inquisitive windlance? You are full of surprises, Master Baggins. Houses warning garden noble innocent silver charm numerous thirsty came Treebeard's warm.",
  },
}

export const Default = Template.bind({})

export const Loading = Template.bind({})
Loading.args = { isLoading: true }

export const Error = Template.bind({})
Error.args = {
  errorMessage: 'ext Grimbold sneak-thief cheekbones myth overfond task',
}
