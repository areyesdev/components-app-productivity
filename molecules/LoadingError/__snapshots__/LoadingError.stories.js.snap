// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Molecules/LoadingError Default 1`] = `
<div
  style={
    Object {
      "alignContent": "flex-start",
      "display": "flex",
      "flexDirection": "column",
      "flexWrap": "wrap",
      "gap": "10px 30px",
      "height": "100%",
      "justifyContent": "flex-start",
      "maxHeight": "auto",
    }
  }
>
  Exile ruins brook Angmar inn wars retaken. Swiftly attacks Grma merely! Opinion Eorlingas inquisitive windlance? You are full of surprises, Master Baggins. Houses warning garden noble innocent silver charm numerous thirsty came Treebeard's warm.
</div>
`;

exports[`Storyshots Molecules/LoadingError Error 1`] = `
<div
  style={
    Object {
      "alignContent": "flex-start",
      "display": "flex",
      "flexDirection": "column",
      "flexWrap": "wrap",
      "gap": "10px 30px",
      "height": "100%",
      "justifyContent": "flex-start",
      "maxHeight": "auto",
    }
  }
>
  <div
    className="error"
  >
    <div
      className="icon-warning icon color-base size-xs background-transparent"
      style={
        Object {
          "height": undefined,
          "width": undefined,
        }
      }
    >
      <svg
        viewBox="0 0 20 20"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          clipRule="evenodd"
          d="M10 20C4.48612 20 0 15.5139 0 10C0 4.48612 4.48612 0 10 0C15.5139 0 20 4.48612 20 10C20 15.5139 15.5139 20 10 20ZM10 1.09864C5.09155 1.09864 1.09864 5.09155 1.09864 10C1.09864 14.9084 5.09155 18.9014 10 18.9014C14.9084 18.9014 18.9014 14.9084 18.9014 10C18.9014 5.09155 14.9084 1.09864 10 1.09864Z"
          fillRule="evenodd"
        />
        <path
          d="M10.7837 11.8579H9.25293L9.08447 4.33594H10.9595L10.7837 11.8579ZM9.0332 14.1577C9.0332 13.8794 9.12354 13.6499 9.3042 13.4692C9.48486 13.2837 9.73145 13.1909 10.0439 13.1909C10.3564 13.1909 10.603 13.2837 10.7837 13.4692C10.9644 13.6499 11.0547 13.8794 11.0547 14.1577C11.0547 14.4263 10.9668 14.6509 10.791 14.8315C10.6152 15.0122 10.3662 15.1025 10.0439 15.1025C9.72168 15.1025 9.47266 15.0122 9.29688 14.8315C9.12109 14.6509 9.0332 14.4263 9.0332 14.1577Z"
        />
      </svg>
    </div>
    <div
      className="spacer horizontal"
      style={
        Object {
          "height": 15,
        }
      }
    />
    <div>
      <p
        className="paragraph color-base size-md weight-normal"
      >
        ext Grimbold sneak-thief cheekbones myth overfond task
      </p>
    </div>
  </div>
</div>
`;

exports[`Storyshots Molecules/LoadingError Loading 1`] = `
<div
  style={
    Object {
      "alignContent": "flex-start",
      "display": "flex",
      "flexDirection": "column",
      "flexWrap": "wrap",
      "gap": "10px 30px",
      "height": "100%",
      "justifyContent": "flex-start",
      "maxHeight": "auto",
    }
  }
>
  <div
    className="container"
  >
    <div
      className="loading is-shown"
    />
  </div>
</div>
`;
